### ISSUE TITLE
Problem with adding new task and changing a column 

### ISSUE THREAD

**Issue by kt919** [kanboard/kanboard#4127]
http://kt919.smarthost.pl/kanboard
admin
haslo12

I have a problem
1. When I try to add new task - I can see it only after refresh page. 
2. When I change column task, moving the tash to another column - I see that something is still loading... 

Please help me. Thank you!

**Comment by creecros** [comment]
Everything worked fine for me.

What Browser are you using?

**Comment by creecros** [comment]
https://projects.nachostudio.ml/
admin
admin

tell me if you have the same issues here.

**Comment by bugwatcher** [comment]
Please, use the [forum](https://kanboard.discourse.group) (https://kanboard.discourse.group) for discussions and feature requests, the bug tracker is reserved only for bugs.
